app/controllers/api/emoji_controller.rb

```diff
@@ -0,0 +1,16 @@
+# this controller returns emoji infomation
+# ex.
+# [
+#   {
+#     "value": "octocat",
+#     "url": "/assets/emoji/octocat-e57ac6c7b8ae639dd53095e4c751cbccd5fa5b74cfaaacc3cfa91a79bf352c54.png"
```

Comment:
__[rubocop]__
* Line is too long. [105/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)